virtualizarr/xarray.py

```diff
@@ -21,6 +21,7 @@ def open_virtual_dataset(
     filepath: str,
     filetype: Optional[str] = None,
     drop_variables: Optional[List[str]] = None,
+    load_variables: Optional[List[str]] = None,
```

Comment:
Not sure "load" is the best name for this kwarg, because these variables will still be lazy. But what else is the opposite of "virtual"? "Real" also has incorrect connotations (and "inline" makes no sense here).

Reply:
Renamed to "loadable", which is marginally better because it doesn't imply that they have already been loaded.